### ISSUE TITLE
I want to apply to tf.layers.con2d

### ISSUE THREAD

**Issue by taki0112** [minhnhat93/tf-SNDCGAN#3]
If I want to apply to tf.layers.con2d, what should I do?
I want to put it in the kernel initializer.

for example
```python
    x = tf.layers.conv2d(inputs=x, filters=channels,
                                      kernel_size=kernel, 
                                      kernel_initializer=spectral_normed_weight(),
                                      strides=stride)
```

Should I just use tf.nn.conv2d... ?

**Comment by patrikohlsson** [comment]
@taki0112 You could use `tf.trainable_variables` and find the kernels, then create the spectral normalization ops.
